packages/light-client/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# @near-js/types
```

Comment:
This will be created automatically by `changeset`, it shouldn't be added manually 🙂